src/CredentialsImport.js

```diff
@@ -51,11 +53,21 @@ class CredentialsImport {
         this.device.uiController?.removeTooltip('interface');
 
         const activeInput = this.device.activeForm?.activeInput;
-        // First blur to make sure we're not already in focus
-        activeInput?.blur();
 
-        // Then focus to open the tooltip
-        activeInput?.focus();
+        const { username, password } = this.device.settings.availableInputTypes.credentials || {};
+        if (this.device.activeForm && activeInput && (username || password)) {
```

Comment:
If you have an `activeInput` you also have an `activeForm` (see line 55), so the first part of this conditional is redundant.